x-pack/plugins/canvas/public/plugin_api.ts

```diff
@@ -18,20 +19,20 @@ export interface CanvasApi {
   addDatasourceUIs: AddToRegistry<any>;
   addElements: AddToRegistry<ElementFactory>;
   addFunctions: AddToRegistry<() => AnyExpressionFunctionDefinition>;
-  addModelUIs: AddToRegistry<any>;
+  addModelUIs: AddToRegistry<ArgumentTypeDefinitionFactory<AnyExpressionFunctionDefinition>>;
```

Comment:
As you use this often, I would shorten to a type in `./types`:

`type AnyArgumentTypeDefinitionFactory = ArgumentTypeDefinitionFactory<AnyExpressionFunctionDefinition>;